prime-router/docs/schema_documentation/direct-cue-covid-19.md

```diff
@@ -1424,7 +1424,7 @@ Custom field. Note, value matched LIVD column "F", "Test Performed LOINC Code"
 
 **PII**: No
 
-**Default Value**: T
+**Default Value**: P
 
```

Comment:
This is the last step in our Onboarding process.

Reply:
Reviewed - moving from Test to Prod